app/logic/Mail/EMail.ts

```diff
@@ -197,11 +200,37 @@ export class EMail extends Message {
 
   async respondToInvitation(response: Responses): Promise<void> {
```

Comment:
This should be somewhere in logic/Calendar/ , if possible. I'm aware that subclasses override this and are dependent on an account. But maybe it could be the calendar, rather than email.

Reply:
Done in e0029db (but I had to tweak the UI)